tf/tfprocess.py

```diff
@@ -279,6 +280,9 @@ def process(self, batch_size, test_batches, batch_splits=1):
                 tf.Summary.Value(tag="Reg term", simple_value=avg_reg_term),
                 tf.Summary.Value(tag="LR", simple_value=self.lr),
                 tf.Summary.Value(tag="MSE Loss", simple_value=avg_mse_loss)])
+            self.net.pb.training_params.learning_rate = self.lr
```

Comment:
Wouldn't it make more sense to include the test values for loss rather than the training values?